components/user/UserLink.js

```diff
@@ -33,7 +33,9 @@ export default function UserLink({ link, username, displayStatsPublic }) {
         color: colors[link.icon],
       }}
     >
-      <Icon name={link.icon} />
+      <span>
+        <Icon name={link.icon} />
+      </span>
```

Comment:
why is the extra `span` needed?

Reply:
This is important because Icons have h-full and w-full so we wrap it inside a span so its not expand same as its parent
container.